lib/components/helpers/transformtype.js

```diff
@@ -0,0 +1,15 @@
+function transformWeatherType(props) {
```

Comment:
@apsitos I would recommend changing the argument name here to something other than 'props' so that it is a bit more specific to what you are pulling in and easier for other developers to understand when reading through the first few times.